pallets/ajuna-awesome-avatars/src/lib.rs

```diff
@@ -1287,7 +1293,7 @@ pub mod pallet {
 				.iter()
 				.map(|id| {
 					let avatar = Self::ensure_ownership(player, id)?;
-					ensure!(avatar.season_id == *season_id, Error::<T>::IncorrectAvatarSeason);
+					ensure!(avatar.season_id == season_id, Error::<T>::IncorrectAvatarSeason);
 					ensure!(
```

Comment:
I would assume this check is not needed since, the encoding of an avatar from the same season must be the same any way since in one season you can only mint one type of encoding